sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2250,6 +2251,22 @@ class HiveDDLSuite
     }
   }
 
+  test("desc formatted table for last access verification") {
+    withTable("t1") {
+      sql(s"create table" +
+        s" if not exists t1 (c1_int int, c2_string string, c3_float float)")
+      val desc = sql("DESC FORMATTED t1").collect().toSeq
+      val lastAccessField = desc.filter((r: Row) => r.getValuesMap(Seq("col_name"))
+        .get("col_name").getOrElse("").equals("Last Access"))
```

Comment:
Can we simplify this via, for instance, `desc.filter($"col_name".startswith("...")).select("data_type")`?